### PR TITLE
Full desktop layout

### PR DESCRIPTION
**Pull request by erezstmn-doit**
Full desktop layout including the Getting started, Spotlight features and learn more sections

### CODE REVIEW

File: website/src/components/ContactSupport/index.js [hunk 1]
```diff
@@ -3,16 +3,18 @@ import styles from './styles.module.css';
...
-      <button href="#" className={styles.contact_button}>
-        raise a ticket
-      </button>
-    </div>
+    <section className={styles.contact_container}>
+      <div className={styles.contact}>
+        <h1 className={styles.contact_title}>Contact support</h1>
+        <p>
+          If you need additional support, one of our knowledgeable support team
+          will be happy to assist.
```
Comment (chaim0m) [inline website/src/components/ContactSupport/index.js:11]: just wondering aloud here if maybe we might want a call to action here something along the lines of 
```suggestion
         If you need additional support, one of our knowledgeable support team
          will be happy to assist.
          Please raise a ticket and we will handle this in no time.
```

Comment (erezstmn-doit) [reply website/src/components/ContactSupport/index.js:11]: These are just mock texts for now. I assume @norosa will take it and edit with the proper content

Comment (norosa) [reply website/src/components/ContactSupport/index.js]: That's correct. Thanks for your vigilance, @chaim0m! I will take this into account when I turn my attention to copywriting.

**Review by norosa — APPROVED** [review]
LGTM. Please squash before merging.

**Pull request merged**